### ISSUE TITLE
For PyTorch Version,  first training step has high loss.

### ISSUE THREAD

**Issue by shoutOutYangJie** [saic-vul/adaptis#17]
For "train_proposals" is False, the loss value is as following:
![image](https://user-images.githubusercontent.com/46734514/72213403-4f626080-3529-11ea-90ff-52a920d268a8.png)

For "train_proposals" is True, the loss value is as following:
![image](https://user-images.githubusercontent.com/46734514/72213412-6608b780-3529-11ea-9f52-0acaa2d9a222.png)

Totally, I think it is still high for segmentation task. But I train ApaptIS by following your training instruction. Have you validated your whether Pytorch version can work or not?
